src/formio/templates/editGridRow.ejs

```diff
@@ -3,9 +3,19 @@
     {% if (ctx.isVisibleInRow(ctx.flattenedComponents[key]) && ctx.flattenedComponents[key].label) { %}
       <div class="utrecht-data-list__item">
         <dt class="utrecht-data-list__item-key">{{ctx.flattenedComponents[key].label}}</dt>
-        <dd class="utrecht-data-list__item-value utrecht-data-list__item-value--html-dd">
-          {{ ctx.getView(ctx.flattenedComponents[key], ctx.row[key]) }}
-        </dd>
+        {% if (typeof ctx.row[key] !== 'object') { %}
+          <dd class="utrecht-data-list__item-value utrecht-data-list__item-value--html-dd">
+            {{ ctx.getView(ctx.flattenedComponents[key], ctx.row[key]) }}
+          </dd>
+        {% } else { %}
+          {% for (const prop in ctx.row[key]) { %}
+            {% if (ctx.row[key][prop] && prop !== 'secretStreetCity') { %}
+              <dd class="utrecht-data-list__item-value utrecht-data-list__item-value--html-dd">
+                {{ ctx.row[key][prop] }}
+              </dd>
+            {% } %}
+          {% } %}
+        {% } %}
```

Comment:
Objects in JS do not formally have any ordering - and we'd like to have a stable presentation order like this:

![image](https://github.com/open-formulieren/open-forms-sdk/assets/5518550/39b05e67-349d-4d8a-8e14-308405021d5e)

I think the `AddressNL` formio component should instead implement the `getView` method: https://github.com/formio/formio.js/blob/4.13.x/src/components/_classes/component/Component.js#L1315 and it looks like as long as you implement `https://github.com/formio/formio.js/blob/4.13.x/src/components/_classes/component/Component.js#L1315` that it should be pretty straight forward. Then you won't need to touch this template at all.